client/db/src/bench.rs

```diff
@@ -155,15 +158,14 @@ impl<B: BlockT> BenchmarkingState<B> {
 	fn add_whitelist_to_tracker(&self) {
 		let mut key_tracker = self.key_tracker.borrow_mut();
 
-		let whitelisted = KeyTracker {
-			has_been_read: true,
-			has_been_written: true,
-		};
-
 		let whitelist = self.whitelist.borrow();
 
 		whitelist.iter().for_each(|key| {
-			key_tracker.insert(key.to_vec(), whitelisted);
+			let whitelisted = KeyTracker {
+				has_been_read: key.has_been_read,
+				has_been_written: key.has_been_written,
+			};
+			key_tracker.insert(key.key.clone(), whitelisted);
```

Comment:
Is there some way to avoid clone here?

Reply:
whitelist is not owned it is borrowed, key is still owned by self.